dist/options/options.js

```diff
@@ -20,10 +20,12 @@ function saveOptions() {
     const exclude = document.getElementById('exclude-radio').checked;
     const labelList = document.getElementById('label-list');
     const labels = labelList.value.split(/[\n]+/).map(s => s.trim()).filter(s => !!s);
+    const bundleDates = document.getElementById('enable-radio').checked;
 
     chrome.storage.sync.set({
         exclude: !!exclude,
         labels: labels,
+        bundleDates: !!bundleDates,
```

Comment:
Suggested name for the setting: `groupMessagesByDate`

(I think of a bundle as a group of messages that share the same label.)